tests/test_project.py

```diff
@@ -772,39 +653,6 @@ def b(job):
             not callable(value) for value in submit_job_operation.directives.values()
         )
 
-    @pytest.mark.filterwarnings("ignore:@FlowProject.operation.with_directives")
-    def test_operation_with_directives(self):
```

Comment:
Does this test need to be updated instead of removed?

Reply:
`with_directives` is just removed so no, since we test directives otherwise.